docs_src/examples/AppServiceExamples.md

```diff
@@ -5,6 +5,7 @@ can be found here in the [edgex-examples](https://github.com/edgexfoundry/edgex-
 
 | <div style="width:250px">Example Name</div>                                                                                                                | Description                                                                                                                                                                    |
 |------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Camera Management](https://github.com/edgexfoundry/edgex-examples/tree/main/application-services/custom/camera-management) | Demonstrates the integration of ONVIF and USB device services with the Edge Video Analytics Microservice for video stream inferencing | 
```

Comment:
I think this sentence can be improvised, probably no need to mention `Edge Video Analytics Microservice` as this is not an Edgex microservice. May be `Demonstrates camera management of ONVIF and USB based devices and also inferencing of their video streams` ?

Reply:
How about this: Utilizes the ONVIF and USB device services and demonstrates the management of these cameras and their integration with video inferencing